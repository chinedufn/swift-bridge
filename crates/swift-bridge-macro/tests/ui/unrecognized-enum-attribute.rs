//! # To Run
//! cargo test -p swift-bridge-macro -- ui trybuild=unrecognized-enum-attribute.rs
// TODO: combine all unrecognized attribute tests

#[swift_bridge::bridge]
mod ffi {
    #[swift_bridge(InvalidAttribute)]
    enum SomeEnum {}
}

fn main() {}
