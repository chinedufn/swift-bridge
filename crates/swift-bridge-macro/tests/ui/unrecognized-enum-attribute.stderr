error: Did not recognize enum attribute "InvalidAttribute".
 --> tests/ui/unrecognized-enum-attribute.rs:7:20
  |
7 |     #[swift_bridge(InvalidAttribute)]
  |                    ^^^^^^^^^^^^^^^^
